llvm/tools/llvm-profgen/ProfileGenerator.cpp

```diff
@@ -492,6 +525,7 @@ void ProfileGenerator::postProcessProfiles() {
   computeSummaryAndThreshold(ProfileMap);
   trimColdProfiles(ProfileMap, ColdCountThreshold);
   calculateAndShowDensity(ProfileMap);
+  filterAmbiguousProfile(ProfileMap);
```

Comment:
nit: move this to be right after trimming, filter is also one form of trimming, and we should do that together before calculating stats/density.